packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -86,6 +95,59 @@ const ShareActivity = WebexPlugin.extend({
     }
   },
 
+  /**
+   * Adds an additional GIF to the share activity
+   * Different from regular add to skip uploading to webex files service
+   * @param {File} gif
+   * @param {Object} options
+   * @param {Object} options.actions
+   * @returns {Promise}
+   */
+  addGif(gif, options) {
+    let gifToAdd = this.gifs.get(gif);
```

Comment:
What are we keying gifs by? Is it an id or unique url? I see we are passing the full gif object around.

Reply:
Ya we do, that's what the regular files do too

Reply:
An entire object is the key yes